src/makevoice/MakeVoice.cpp

```diff
@@ -137,11 +142,11 @@ HRESULT install()
         CComPtr<ISpDataKey> cpDataKeyAttribs;
         hr = SpCreateNewTokenEx(
             SPCAT_VOICES,
-            TEXT(AUTOMATION_VOICE_ID),
+            L"" AUTOMATION_VOICE_ID,
```

Comment:
I replaced `TEXT` uses here with `L""` preprocess concatenation here because as far as I can tell `SpCreateNewTokenEx` only takes `wchar_t` arguments.